### DISCUSSION TITLE
Is H.265 compatible with "webrtc" or "mse", I can only get an image with "jsmpeg"?

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#4908] [Ask A Question] [answered]
Is H.265 compatible with "webrtc" or "mse", I can only get an image with "jsmpeg"? 

Thank you.

**Comment by NickM-27** [comment, marked as answer]
MSE supports h.265 but it also needs to be supported by the browser, Firefox doesn't support h.265. 

WebRTC does not support h.265 except in Safari

**Reply by Red-ington** [reply]
more & more camera's are now supporting H265, looks like we need more support going forward.  makes smaller files on my server recordings that h264 equivalent 4k camera's

**Reply by NickM-27** [reply]
> more & more camera's are now supporting H265, looks like we need more support going forward.  makes smaller files on my server recordings that h264 equivalent 4k camera's 

Sure, but that has nothing to do with frigate. WebRTC as a technology does not support h265, nothing frigate can do about that.

**Reply by Red-ington** [reply]
what else can be used for H265 with the same advantages.

**Reply by NickM-27** [reply]
> what else can be used for H265 with the same advantages. 

If you use a browser other than Firefox mse supports h265
